src/main/java/edu/iis/powp/adapter/PlotterToDrawControllerAdapter.java

```diff
@@ -9,12 +12,14 @@
 /**
  * Plotter adapter to drawer with several bugs. 
  */
-public class MyAdapter extends DrawPanelController implements IPlotter
+public class PlotterToDrawControllerAdapter extends DrawPanelController implements IPlotter
 { 
 	private int startX = 0, startY = 0;
+	private DrawPanelController drawPanelController = null;
 	
-    public MyAdapter() {
+    public PlotterToDrawControllerAdapter() {
 		super();
+		this.drawPanelController = Application.getComponent(DrawPanelController.class);
```

Comment:
1. Constructor injection should be used (using a parameter). This is unnecessary direct coupling with Applicationclass.
2. Inheritance is not used (no super drawLine calls). Should be removed!